crates/bins/src/bin/datadog-static-analyzer.rs

```diff
@@ -471,7 +471,11 @@ fn main() -> Result<()> {
     let mut number_of_rules_used = 0;
     // Finally run the analysis
     for language in &languages {
-        let files_for_language = filter_files_for_language(&files_to_analyze, language);
+        let mut files_for_language = filter_files_for_language(&files_to_analyze, language);
+        if language == &Language::JavaScript {
+            files_for_language =
+                filter_out_minified_files(&files_for_language, directory_path, use_debug);
+        }
```

Comment:
1. Implement this rather as the `is_generated_file`. Look at [this](https://github.com/DataDog/datadog-static-analyzer/blob/main/crates/static-analysis-kernel/src/analysis/analyze.rs#L186) and we should implement something similar.
2. This should be enabled only when `analysis_option.ignore_generated_files` is true (similar to (1))